src/authentication/store/actions.tsx

```diff
@@ -103,9 +112,9 @@ export const setLoginError = (): SetLoginErrorAction => ({
 	error: true,
 });
 
-export const setLoginLoading = (): SetLoginLoadingAction => ({
+export const setLoginLoading = (isLoading: boolean): SetLoginLoadingAction => ({
```

Comment:
Als je het hier `loading` noemt, kan je hieronder `loading: isLoading,` vervangen door `loading,`.

Reply:
nee want er is al een globale variabele die loading heet. Dan zou het onduidelijk zijn of ik de functie parameter bedoel of the globale variabele